CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Drop the external initramfs and only use the initramfs bundled in the kernel [Florin]
```

Comment:
As far as I know it was always this way. This how the deployment job was designed.